### DISCUSSION TITLE
Why when I right click to save a image or try to do a video download do I get a Failed-Network Error

### DISCUSSION THREAD

**Discussion by Dentonknifeworks** [aaronwmorris/indi-allsky#933] [Q&A] [answered]
![Screenshot 2023-09-07 at 11 56 06 PM](https://github.com/aaronwmorris/indi-allsky/assets/64991600/579c810a-2c61-4a8b-86b3-f36f1c9989ce)

**Comment by aaronwmorris** [comment]
indi-allsky usually does not serve the images and videos directly.  Those are served directly by the web server, usually Apache.

You may check the web server logs at `/var/log/apache2/error.log` and `/var/log/apache2/other_vhosts_access.log`.

**Reply by Dentonknifeworks** [reply]
Found that there is no issue using Safari. The issue looks to be Chrome. No errors in the logs that you suggested.

**Reply by Dentonknifeworks** [reply]
Also found FireFox works. Chrome on my MAC or PC is not downloading. So seems to be Chrome and the Apache server that is on the PI that are not getting along. Are you using Chrome? Anyone else seeing this problem?

**Reply by aaronwmorris** [reply]
I am seeing the same error when using Chrome on Linux.

I have had issues with Chrome and the self-signed certificate in the past.  In those cases, I could not even open the website because there was no way to approve (even temporarily) the certificate.

**Reply by Vertikar** [reply]
I've run into the issue before in Chrome on Windows too, switching to plain http worked.
Didn't try Firefox.

**Reply by aaronwmorris** [reply, marked as answer]
I found the fix.  https://support.google.com/chrome/thread/10630812/how-to-download-a-file-in-spite-of-error-failed-bad-certificate?hl=en

1. Click the exclamation icon next to the URL bar
2. Click site settings
3. Set 'automatic downloads' to Allow
4. Refresh page and download the document

**Reply by Dentonknifeworks** [reply]
Yes that seems to do it.
